concent_api/core/decorators.py

```diff
@@ -0,0 +1,328 @@
+from functools import wraps
+from logging import getLogger
+from typing import Any
+from typing import Callable
+from typing import Union
+
+from django.conf import settings
+from django.db import transaction
+from django.http import HttpRequest
+from django.http import HttpResponse
+from django.http import HttpResponseNotAllowed
+from django.http import JsonResponse
+from golem_messages import dump
+from golem_messages import message
+from golem_messages.exceptions import FieldError
+from golem_messages.exceptions import MessageError
+from golem_messages.exceptions import MessageFromFutureError
+from golem_messages.exceptions import MessageTooOldError
+from golem_messages.exceptions import TimestampError
+
+from common import logging
+from common.constants import ERROR_IN_GOLEM_MESSAGE
+from common.constants import ErrorCode
+from common.exceptions import ConcentBaseException
+from common.exceptions import ConcentInSoftShutdownMode
+from common.exceptions import ConcentValidationError
+from common.helpers import join_messages
+from common.logging import get_json_from_message_without_redundant_fields_for_logging
+from common.logging import log
+from common.logging import log_400_error
+from common.shortcuts import load_without_public_key
+from core.exceptions import CreateModelIntegrityError
+from core.exceptions import UnsupportedProtocolVersion
+from core.exceptions import NonPositivePriceTaskToComputeError
+from core.utils import is_given_golem_messages_version_supported_by_concent
+from core.validation import get_validated_client_public_key_from_client_message
+from core.validation import is_golem_message_signed_with_key
+
+logger = getLogger(__name__)
+
+
+def validate_protocol_version_in_core(view: Callable) -> Callable:
+    @wraps(view)
+    def wrapper(
+        request: HttpRequest,
+        client_message: message.Message,
+        client_public_key: bytes,
+        *args: list,
```

Comment:
Isn't mypy complaining about this? From what I remember `*args` and `**kwargs` was working with `Any` type, as they can be anything. But if it is not complaining than ok.